packages: init

MarcelCoding opened the pull request:
- [ ] link open PRs ``https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+python3Packages.cryptography+in%3Atitle``
- [x] matrix preview picture
- [x] sort results by levenshtein
- [x] purl
	- [x] picked https://redirect.github.com/NixOS/nixpkgs/pull/454333
	- [x] collect metadata
	- [x] render
- [x] use and show disabled field
- [ ] platforms (x86_64, aarch64, Linux, Darwin) => #291
	- [ ] collect metadata
	- [ ] render
- [x] lz4_fast compression https://github.com/NuschtOS/ixx/issues/75
- [x] fix clang, zoom, immich, opens packages.openssh
	- [x] locally
	- [x] get it deployed
- [ ] Light theme red error box => #292
- [x] link package options to package search => #293
- [ ] custom markdown parser for custom NixOS syntax https://docs.rs/comrak/latest/comrak/macro.create_formatter.html => #297
- [x] wildcards at the end of labels do not work
- [x] teams
- [x] longDescription
- [x] meta.sourceProvenance => #294
- [x] meta.downloadPage => #295
- [x] meta.changelog => #296

MarcelCoding committed:
nix: try to eval less

MarcelCoding committed:
nix: fix eval

MarcelCoding committed:
nix: redurce number if groups

MarcelCoding committed:
nix: try to redurce mem usage

MarcelCoding committed:
nix: handle eval error

MarcelCoding committed:
nix: fix license

MarcelCoding committed:
nix: split eval of pkg data into separate process

MarcelCoding committed:
nix: replace ln with cp

MarcelCoding committed:
nix: fix partitions file

MarcelCoding committed:
nix: debug

MarcelCoding committed:
nix: optimize

MarcelCoding committed:
nix: fix lib

MarcelCoding committed:
nix: redurce group count

MarcelCoding committed:
nix: update nixpkgs

MarcelCoding committed:
nix: update lock file

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/0ac51146f1d1e9f6d5276fb6f0707c1305514a53?narHash=sha256-cvMs2G835YboC64hwwo2kaP6yIvCrzDCoMZsBT3rRzQ%3D' (2025-11-05)
  → 'github:NuschtOS/ixx/e07f0944d83d9ef30821cfd1e2eecd245b531420?narHash=sha256-DcXVvapKCvBQcmqT3e5Mh0ZIsIv7BSYH2qhVyROrG/o%3D' (2025-11-06)

SuperSandro2000 committed:
Remove fixed TODO

by using the first name of the package instead of the package set this
is no longer a concern.

SuperSandro2000 committed:
Fix inherit

SuperSandro2000 committed:
Add declaration to packages

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/e07f0944d83d9ef30821cfd1e2eecd245b531420?narHash=sha256-DcXVvapKCvBQcmqT3e5Mh0ZIsIv7BSYH2qhVyROrG/o%3D' (2025-11-06)
  → 'github:NuschtOS/ixx/b39b9cd50830087599babeac1cf38cdb77d72a00?narHash=sha256-KY%2BKHP7zZn9%2BW57vhOkQm5YFEDbTFBxP/hVi2LPXKGM%3D' (2025-11-07)

SuperSandro2000 committed:
Fix maintainers and teams

SuperSandro2000 committed:
Add licenses and maintainers mappings

SuperSandro2000 committed:
Convert githubId to string

SuperSandro2000 committed:
Fix teams eval when someone did something weird

SuperSandro2000 committed:
Sort, run offline

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/b39b9cd50830087599babeac1cf38cdb77d72a00?narHash=sha256-KY%2BKHP7zZn9%2BW57vhOkQm5YFEDbTFBxP/hVi2LPXKGM%3D' (2025-11-07)
  → 'github:NuschtOS/ixx/fe7235c63f4ef87cbcd1dabfc4c65bd1302edd04?narHash=sha256-AtnNSU093BgK4pHf1l9YTa3ABZOsZ3cDAR4x9v4MMKM%3D' (2025-11-08)

MarcelCoding committed:
package: add broken, insecure and eval error notices

SuperSandro2000 committed:
Replace insecure with known_vulnerabilities

SuperSandro2000 committed:
Fix

SuperSandro2000 committed:
Drop unfree

SuperSandro2000 committed:
Add mkCollectManDerivations

MarcelCoding committed:
package: render known_vulnerabilities as html

SuperSandro2000 committed:
Fix man build

SuperSandro2000 committed:
Filter out none existing packages

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/cebe312dcfefde35f93532584701cc5fa9c1f236?narHash=sha256-xbmumVRh9O%2BxML4rg7zDDHqThUWl2gtR6EHErIwrBEY%3D' (2025-11-05)
  → 'github:NixOS/nixpkgs/5082a6ff9afb880bec2d1e915d5614f07a2aaf0a?narHash=sha256-wAp020NyGm73eQTuq3yZuD%2BvG/1d9fvBbaKk%2BJfgZtM%3D' (2025-11-08)

SuperSandro2000 committed:
flake:

MarcelCoding committed:
package: add decleration

SuperSandro2000 committed:
Improve mappings

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/dd61cda64e6e6ac48b772287545619dfa406c86d?narHash=sha256-LdBgvYw5nizOMnHbBCvaDhCWCkxJBKsL4c5brZDj4Rc%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/43898294162913be1bf1c1a5d52235f9c40958f5?narHash=sha256-0rI4z6X2S0xku7b3UGrfejuqn7Gw24MUT6o7/vug3Hs%3D' (2025-11-08)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/cebe312dcfefde35f93532584701cc5fa9c1f236?narHash=sha256-xbmumVRh9O%2BxML4rg7zDDHqThUWl2gtR6EHErIwrBEY%3D' (2025-11-05)
  → 'github:NuschtOS/nuschtpkgs/ae814fd3904b621d8ab97418f1d0f2eb0d3716f4?narHash=sha256-YRqMDEtSMbitIMj%2BJLpheSz0pwEr0Rmy5mC7myl17xs%3D' (2025-11-05)

SuperSandro2000 committed:
flake:

MarcelCoding committed:
update to ixx changes

SuperSandro2000 committed:
Keep githubId as int

SuperSandro2000 committed:
build-packages: cleanup

SuperSandro2000 committed:
Find man pages in strange places

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/fca8b55714c922e4c8062c1daeadecf797dba51b?narHash=sha256-PUumKjx7E6JJInA8fsM61XbmRwnQByogReRU1EuBgW8%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/2d5e76faf082878ae4536996ad0b3954276188ab?narHash=sha256-m0MZ7ipB1CPeyEpsysf3psTB9SbqH2g2vWKZR2rAvKI%3D' (2025-11-08)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/ae814fd3904b621d8ab97418f1d0f2eb0d3716f4?narHash=sha256-YRqMDEtSMbitIMj%2BJLpheSz0pwEr0Rmy5mC7myl17xs%3D' (2025-11-05)
  → 'github:NuschtOS/nuschtpkgs/a10c9c0a5a2867f1d451b78f3b4f0366213528c4?narHash=sha256-jJMH1pZYmzo5qhVbxO1QlFiwECf28/VLatVLfPP0wKY%3D' (2025-11-08)

MarcelCoding committed:
search: add licenses and maintainers

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/a10c9c0a5a2867f1d451b78f3b4f0366213528c4?narHash=sha256-jJMH1pZYmzo5qhVbxO1QlFiwECf28/VLatVLfPP0wKY%3D' (2025-11-08)
  → 'github:NuschtOS/nuschtpkgs/4ed34a26057d00ebcdb0b482f7ca3971f8a519ba?narHash=sha256-cSEwjyEd5yka78yppLkrXYmiioFUnwrc3eakSETeF7E%3D' (2025-11-08)

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/4ed34a26057d00ebcdb0b482f7ca3971f8a519ba?narHash=sha256-cSEwjyEd5yka78yppLkrXYmiioFUnwrc3eakSETeF7E%3D' (2025-11-08)
  → 'github:NuschtOS/nuschtpkgs/42bd652ebe09d75b8a8d626d1d6b29effcb7efb8?narHash=sha256-3GNJgKQjE/6d%2BB6G4%2BzMF6CGZp2RcNGSWFQt7DeqXNk%3D' (2025-11-08)

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/2d5e76faf082878ae4536996ad0b3954276188ab?narHash=sha256-m0MZ7ipB1CPeyEpsysf3psTB9SbqH2g2vWKZR2rAvKI%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/f4926bfb56d21d110d66cbaa549af230d4707148?narHash=sha256-VsSSq%2Beu3R9F0e0zjj3MAFk4y65ufXvPiQg9H4iUcMg%3D' (2025-11-08)

SuperSandro2000 committed:
wrapper: build all

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/f4926bfb56d21d110d66cbaa549af230d4707148?narHash=sha256-VsSSq%2Beu3R9F0e0zjj3MAFk4y65ufXvPiQg9H4iUcMg%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/3b3edd00fcb40ccdc6c42387885174d6b65642d5?narHash=sha256-E%2BITLNvo2SkjC3IlL1V69%2B8kU2Hu44b7v32sRGQM5FQ%3D' (2025-11-08)

SuperSandro2000 committed:
First successful build

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/3b3edd00fcb40ccdc6c42387885174d6b65642d5?narHash=sha256-E%2BITLNvo2SkjC3IlL1V69%2B8kU2Hu44b7v32sRGQM5FQ%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/93679d1c0c87175f484c1c742f43c44e94c643f7?narHash=sha256-E9QZeneaksZ%2B0h1y0N%2BF9VL1/6JFmVRIzshbtnd900E%3D' (2025-11-08)

MarcelCoding committed:
search: handle errors

SuperSandro2000 committed:
man: filter wrappers

MarcelCoding committed:
nix: migrate config into json

MarcelCoding committed:
router: make options and packages optional

MarcelCoding committed:
pages: build pages with packages

MarcelCoding committed:
meta: fetch only once

MarcelCoding committed:
meta: fetch only once

MarcelCoding committed:
package: hide maintainers if empty

MarcelCoding committed:
package: remove cookie third party cookies

SuperSandro2000 committed:
Sort things

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/93679d1c0c87175f484c1c742f43c44e94c643f7?narHash=sha256-E9QZeneaksZ%2B0h1y0N%2BF9VL1/6JFmVRIzshbtnd900E%3D' (2025-11-08)
  → 'github:NuschtOS/ixx/b08bc0b533e1bd92ad17d1ae5867a5fd64619a26?narHash=sha256-FIMK/YmdpohEE2SDlzvLX9jS6vZCpi03N8B4yC%2B51Vk%3D' (2025-11-09)
• Updated input 'nix-index-database':
    'github:Mic92/nix-index-database/359ff6333a7b0b60819d4c20ed05a3a1f726771f?narHash=sha256-Pu1v3mlFhRzZiSxVHb2/i/f5yeYyRNqr0RvEUJ4UgHo%3D' (2025-11-02)
  → 'github:Mic92/nix-index-database/15c5451c63f4c612874a43846bfe3fa828b03eee?narHash=sha256-C9F1C31ys0V7mnp4EcDy7L1cLZw/sCTEXqqTtGnvu08%3D' (2025-11-09)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/42bd652ebe09d75b8a8d626d1d6b29effcb7efb8?narHash=sha256-3GNJgKQjE/6d%2BB6G4%2BzMF6CGZp2RcNGSWFQt7DeqXNk%3D' (2025-11-08)
  → 'github:NuschtOS/nuschtpkgs/d8c4d56601c4343609a4ead977d357fa11136c2f?narHash=sha256-enFFO3ryrFqxmf7raQ%2BpwsRlVvDQiNtUJL0KX7IVcxI%3D' (2025-11-09)

MarcelCoding committed:
package: fix maintainer images

SuperSandro2000 committed:
Sort pname, name, version again

SuperSandro2000 committed:
Do not display out output alone

Every package has that

SuperSandro2000 committed:
Move things around

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/b08bc0b533e1bd92ad17d1ae5867a5fd64619a26?narHash=sha256-FIMK/YmdpohEE2SDlzvLX9jS6vZCpi03N8B4yC%2B51Vk%3D' (2025-11-09)
  → 'github:NuschtOS/ixx/1bd5e4ce4cec8eb8669b0f97c61f098268811981?narHash=sha256-YfJNRTLoRTJvSKQkVO6EugKkbmKpHroNMWiPQPKBQo0%3D' (2025-11-09)

MarcelCoding committed:
html: fix syntax

MarcelCoding committed:
fixx: update

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/d76d2cc7f8b05fc9038b0416af583a5a5dd41d46?narHash=sha256-b%2BwpJLAX0OZ5orzDawSxSDPTAAfcJvc4MRzDMHvsMPU%3D' (2025-11-10)
  → 'github:NuschtOS/ixx/6f3fdf7e07ae4cd678317f503fda41e29d9e67dc?narHash=sha256-wByRLD0bCmLS0cWN1BA%2BAyhwU%2BEhj0RWZYI0Khh8wck%3D' (2025-11-10)

MarcelCoding committed:
misc: update; print index size

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/6f3fdf7e07ae4cd678317f503fda41e29d9e67dc?narHash=sha256-wByRLD0bCmLS0cWN1BA%2BAyhwU%2BEhj0RWZYI0Khh8wck%3D' (2025-11-10)
  → 'github:NuschtOS/ixx/2a10bdc11d7460757966f12aa5f2ac6e5c9c484c?narHash=sha256-1QvV2vkkTxgB7gKkkp5UHNBKEEnZP8bA/uYn3McCMxo%3D' (2025-11-10)

MarcelCoding committed:
fixx: update

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/2a10bdc11d7460757966f12aa5f2ac6e5c9c484c?narHash=sha256-1QvV2vkkTxgB7gKkkp5UHNBKEEnZP8bA/uYn3McCMxo%3D' (2025-11-10)
  → 'github:NuschtOS/ixx/80549ae7af1d4cea01b046b7496399fa11b6d574?narHash=sha256-JmxQwMhQSTvcuSFYqbs6qxCjHnIYeA3ephR8Au7DLUM%3D' (2025-11-10)

MarcelCoding committed:
ixx: improve directory layout

MarcelCoding committed:
ci: add cache

SuperSandro2000 commented on nix/wrapper.nix:
```diff
@@ -208,7 +208,7 @@ rec {
       config = {
         inherit baseHref title;
-        dataBase = baseHref;
+        dataBase = "${baseHref}data/";
```
Do we somewhere enforce baseHref to end in a / ?

MarcelCoding committed:
data: fix urls

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/80549ae7af1d4cea01b046b7496399fa11b6d574?narHash=sha256-JmxQwMhQSTvcuSFYqbs6qxCjHnIYeA3ephR8Au7DLUM%3D' (2025-11-10)
  → 'github:NuschtOS/ixx/8abaabdb586ba065391bd5acf5b31a421d5468e8?narHash=sha256-ItV0VvAd8mhP85MxdBMIX26a5LSaO4tHL3ctZSHIRHA%3D' (2025-11-10)

MarcelCoding committed:
fixx: update

SuperSandro2000 committed:
Use backports branch to get nix 2.32 on stable

SuperSandro2000 committed:
Add CPE

SuperSandro2000 committed:
Remove whitespace highlight in maintainers

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/88284e0b9fcb3b1f05bf9eb3198babee6ff3a708?narHash=sha256-cMRRnnknfMlDhbscryLr0UYy%2Byd01i2YnEIU8j7OOFM%3D' (2025-11-10)
  → 'github:NuschtOS/ixx/c6fc4edb243da241883c50db8b722420c946d516?narHash=sha256-F1aTk/QVVRCaRsNBxVEcBSPAkxQrWafk4slfyFRDwsw%3D' (2025-11-11)

MarcelCoding committed:
mixc: update

MarcelCoding committed:
fixx: update

MarcelCoding committed:
fixx: update

MarcelCoding committed:
header: fix option and package count

SuperSandro2000 committed:
Only show Possible Cpes if there any

SuperSandro2000 committed:
Bring back gitAndTools and haskell

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/c6fc4edb243da241883c50db8b722420c946d516?narHash=sha256-F1aTk/QVVRCaRsNBxVEcBSPAkxQrWafk4slfyFRDwsw%3D' (2025-11-11)
  → 'github:NuschtOS/ixx/8c9b065e397b35fc1e37b66f7e21203ca261e66a?narHash=sha256-Q0qZTJv0XHHJlFLmY2UDkrGzrtyNzGDzKa2y3Idjjjc%3D' (2025-11-11)

SuperSandro2000 committed:
Pass through data and config

MarcelCoding committed:
wrapper: deupe haskell

MarcelCoding committed:
package: render option description as html

MarcelCoding committed:
table: top align content

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/8c9b065e397b35fc1e37b66f7e21203ca261e66a?narHash=sha256-Q0qZTJv0XHHJlFLmY2UDkrGzrtyNzGDzKa2y3Idjjjc%3D' (2025-11-11)
  → 'github:NuschtOS/ixx/4779aa5b0e00ec77391e4436609c4ca90327c59d?narHash=sha256-ui20nGBK1K6fexTKN8k0I/rkbxOxO3BijYjZtKvt4yA%3D' (2025-11-11)

SuperSandro2000 committed:
Make local debugging of mkPackage possible

MarcelCoding committed:
packages: top align columns

SuperSandro2000 committed:
Add disabled

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/4779aa5b0e00ec77391e4436609c4ca90327c59d?narHash=sha256-ui20nGBK1K6fexTKN8k0I/rkbxOxO3BijYjZtKvt4yA%3D' (2025-11-11)
  → 'github:NuschtOS/ixx/bde6918aa38c96e9dc65bf64dd967b31ac2322a8?narHash=sha256-nRD8Oc4%2B4We%2BepY%2BEp2uB9x3s/ahy9Kx5/KBdp/qIUo%3D' (2025-11-12)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/d8c4d56601c4343609a4ead977d357fa11136c2f?narHash=sha256-enFFO3ryrFqxmf7raQ%2BpwsRlVvDQiNtUJL0KX7IVcxI%3D' (2025-11-09)
  → 'github:NuschtOS/nuschtpkgs/6dd888263304a3e4c18eaa80bfa44d1b97639173?narHash=sha256-Yy%2BmFCUs6L3iMkXWfoI4g/6QQaut58QTE2Vb3kxutX8%3D' (2025-11-12)

SuperSandro2000 committed:
Display pname/version more predictable

SuperSandro2000 committed:
Filter more haskell things

MarcelCoding committed:
ci: redurce max jobs to not run out of memory

MarcelCoding committed:
ixx: update; add code block to cpe

SuperSandro2000 committed:
Improve debugging

SuperSandro2000 committed:
Add matrix pictures

SuperSandro2000 committed:
Drop console.logs

SuperSandro2000 committed:
Fix typos

SuperSandro2000 committed:
Fix unused imports

SuperSandro2000 committed:
Catch errors when fetching matrix

SuperSandro2000 committed:
Downscale matrix pictures, improve rendering

SuperSandro2000 committed:
Fix minor things

SuperSandro2000 committed:
Fix meta.homepage hack

SuperSandro2000 committed:
Cache matrix result

SuperSandro2000 committed:
Display disabled, improve wording

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/cf46014f9065d08574b53133a8a85b863cd0be42?narHash=sha256-3gGD826WHk/1Puf5HFGdyAbxr63Rl%2B/wIaZHJTgJRxo%3D' (2025-11-12)
  → 'github:NuschtOS/ixx/f34f5666dac22009c72bac3371b75c5c8cafe450?narHash=sha256-QfMFfFDzYF/kzMDF8evxH5qTQKp/ripIRIWopN8DI9Q%3D' (2025-11-12)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/6dd888263304a3e4c18eaa80bfa44d1b97639173?narHash=sha256-Yy%2BmFCUs6L3iMkXWfoI4g/6QQaut58QTE2Vb3kxutX8%3D' (2025-11-12)
  → 'github:NuschtOS/nuschtpkgs/955c9fd38c8ac31e9bd1aed184dcf6f08335aef3?narHash=sha256-um21N%2BwKb1K8LCGBoWkNGyx9A2392Zlh8lumLpJ/f/o%3D' (2025-11-12)

SuperSandro2000 committed:
Parse metadata with mixed maintainers & teams better

SuperSandro2000 committed:
Comment why tests is disabled

SuperSandro2000 committed:
Comment vm-variant

SuperSandro2000 committed:
Next maintainers fix up

SuperSandro2000 committed:
Evaluate nixosTests, drop scope

SuperSandro2000 committed:
Fix teams eval

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/f34f5666dac22009c72bac3371b75c5c8cafe450?narHash=sha256-QfMFfFDzYF/kzMDF8evxH5qTQKp/ripIRIWopN8DI9Q%3D' (2025-11-12)
  → 'github:NuschtOS/ixx/1088de423f0656fe893077dae5c0eb0079b0d066?narHash=sha256-Vn03qRzOY4XQnYSwj7YwQUCZXnm0HaaRVS7K/3Z7GaQ%3D' (2025-11-12)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/955c9fd38c8ac31e9bd1aed184dcf6f08335aef3?narHash=sha256-um21N%2BwKb1K8LCGBoWkNGyx9A2392Zlh8lumLpJ/f/o%3D' (2025-11-12)
  → 'github:NuschtOS/nuschtpkgs/b3fe9660b8b1a456bc68d0cac35dff1cd133fc3e?narHash=sha256-RA4t6zqWDdu1mySiOEc6z3mHRLYNXHAtcmHR2ln/4BY%3D' (2025-11-13)

MarcelCoding committed:
maintainer: add alt img

MarcelCoding committed:
package: refactor maintainer

MarcelCoding committed:
package: execute querys async

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/1088de423f0656fe893077dae5c0eb0079b0d066?narHash=sha256-Vn03qRzOY4XQnYSwj7YwQUCZXnm0HaaRVS7K/3Z7GaQ%3D' (2025-11-12)
  → 'github:NuschtOS/ixx/69ae215b8c7f13f3e216ec7a10c950345f570995?narHash=sha256-weXaZV41xwwolPWljI59ElQ7S2PURTJSwhs7heIi%2BSE%3D' (2025-11-14)

SuperSandro2000 committed:
Use current system

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/b3fe9660b8b1a456bc68d0cac35dff1cd133fc3e?narHash=sha256-RA4t6zqWDdu1mySiOEc6z3mHRLYNXHAtcmHR2ln/4BY%3D' (2025-11-13)
  → 'github:NuschtOS/nuschtpkgs/9475504f5c3f37d7711bf944f5b1a87f2b0446bc?narHash=sha256-OXHVbeROkMApJu03wc4yN/RXw3UG2xn82uJnpcYv5XM%3D' (2025-11-14)

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/9475504f5c3f37d7711bf944f5b1a87f2b0446bc?narHash=sha256-OXHVbeROkMApJu03wc4yN/RXw3UG2xn82uJnpcYv5XM%3D' (2025-11-14)
  → 'github:NuschtOS/nuschtpkgs/2f89ccf8b2b6c044d7b4824882c567f892f381af?narHash=sha256-rwhf4Og0f79Jy8w395WhbarFZs7dMQFtpFeuJWA98dc%3D' (2025-11-14)

MarcelCoding committed:
ci: build agains forked nixpkgs

MarcelCoding committed:
ci: redurce amount of parallel jobs

MarcelCoding committed:
wrapper: generate smaller batches

SuperSandro2000 committed:
Make direnv-instant work

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/69ae215b8c7f13f3e216ec7a10c950345f570995?narHash=sha256-weXaZV41xwwolPWljI59ElQ7S2PURTJSwhs7heIi%2BSE%3D' (2025-11-14)
  → 'github:NuschtOS/ixx/4edb8d04a1b137dadeecf8cac8091c0467266452?narHash=sha256-CMctioQzgGnDZ29uKux4F%2BqVx6zi%2BBpQoEwseKUPWbI%3D' (2025-11-27)

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/4edb8d04a1b137dadeecf8cac8091c0467266452?narHash=sha256-CMctioQzgGnDZ29uKux4F%2BqVx6zi%2BBpQoEwseKUPWbI%3D' (2025-11-27)
  → 'github:NuschtOS/ixx/39344b472282ae1ce665a00d23c91e975878d84d?narHash=sha256-wElrpYdfp0tF7GUuwWjlYtKMGwTjQ6PcNbe1Pz7tjts%3D' (2025-11-27)
• Updated input 'nix-index-database':
    'github:Mic92/nix-index-database/15c5451c63f4c612874a43846bfe3fa828b03eee?narHash=sha256-C9F1C31ys0V7mnp4EcDy7L1cLZw/sCTEXqqTtGnvu08%3D' (2025-11-09)
  → 'github:Mic92/nix-index-database/d7423982c7a26586aa237d130b14c8b302c7a367?narHash=sha256-NPyc76Wxmv/vAsXJ8F%2B/8fXECHYcv2YGSqdiSHp/F/A%3D' (2025-11-23)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/2f89ccf8b2b6c044d7b4824882c567f892f381af?narHash=sha256-rwhf4Og0f79Jy8w395WhbarFZs7dMQFtpFeuJWA98dc%3D' (2025-11-14)
  → 'github:NuschtOS/nuschtpkgs/c5c694ccd67705b46a024c51734cfacc098d891a?narHash=sha256-9Td9xBdmU7lKb0EdfNXc0hFfQ/J8KnSocmQshEPkmwc%3D' (2025-11-25)

MarcelCoding committed:
misc: envs.sh shutdown

SuperSandro2000 committed:
Try to fix CI on none x86_64-linux hosts

SuperSandro2000 commented:
https://redirect.github.com/NixOS/nixpkgs/pull/465607

SuperSandro2000 committed:
Limit max jobs to 2

SuperSandro2000 committed:
Reduce trace

SuperSandro2000 committed:
Fix download

SuperSandro2000 committed:
Test against new stable

SuperSandro2000 committed:
Collect purls

SuperSandro2000 committed:
Add long_description

SuperSandro2000 committed:
Fix syntax

SuperSandro2000 committed:
Update hash

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/c26292b4a57a25bd3f20221ac5e87202073a1b74?narHash=sha256-IkCiJwzwKVO4EwI94DN%2BMdpVSmbkif8ca5Se0d2oEtI%3D' (2025-11-27)
  → 'github:NuschtOS/ixx/986d0fbdb36c295d21d5f97f11d3fdbf7f02256c?narHash=sha256-cF76f/bvBVR7Un0DyXBycNtgkdbHeWPxDaqEIP/4r34%3D' (2025-11-30)

SuperSandro2000 committed:
Fix hash

SuperSandro2000 committed:
Fix syntax

SuperSandro2000 committed:
Revert hash

SuperSandro2000 committed:
Not every account has an avatar

SuperSandro2000 commented on src/app/core/components/package/package.component.html:
```diff
@@ -0,0 +1,196 @@
+          <td [innerHTML]="package.description"></td>
+        </tr>
+        }
+        @if (package.long_description) {
```
This should be longDescription

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/986d0fbdb36c295d21d5f97f11d3fdbf7f02256c?narHash=sha256-cF76f/bvBVR7Un0DyXBycNtgkdbHeWPxDaqEIP/4r34%3D' (2025-11-30)
  → 'github:NuschtOS/ixx/fbc470214fdff2fdce544c5e7aaace06ad722102?narHash=sha256-Wq/mQ0VjIbRKbrum0nI2Ox0cJ6sJOeoXVpP0osqQRpM%3D' (2025-11-30)
• Updated input 'nix-index-database':
    'github:Mic92/nix-index-database/d7423982c7a26586aa237d130b14c8b302c7a367?narHash=sha256-NPyc76Wxmv/vAsXJ8F%2B/8fXECHYcv2YGSqdiSHp/F/A%3D' (2025-11-23)
  → 'github:Mic92/nix-index-database/5a3ff8c1a09003f399f43d5742d893c0b1ab8af0?narHash=sha256-77jL5H5x51ksLiOUDjY0ZK8e2T4ZXLhj3ap8ETvknWI%3D' (2025-11-30)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/c5c694ccd67705b46a024c51734cfacc098d891a?narHash=sha256-9Td9xBdmU7lKb0EdfNXc0hFfQ/J8KnSocmQshEPkmwc%3D' (2025-11-25)
  → 'github:NuschtOS/nuschtpkgs/741dabf48e7effc1cc6a43ed97d071214ed2cc31?narHash=sha256-tSbNnYn/gB6DDhZ8ursugXBIMxZ64zuveD517ZpXYZs%3D' (2025-12-01)

SuperSandro2000 committed:
Update purl config flag

SuperSandro2000 committed:
Fix long description

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/fbc470214fdff2fdce544c5e7aaace06ad722102?narHash=sha256-Wq/mQ0VjIbRKbrum0nI2Ox0cJ6sJOeoXVpP0osqQRpM%3D' (2025-11-30)
  → 'github:NuschtOS/ixx/0b49397b3c1160944687aba813799f3f41fe6500?narHash=sha256-henej5X7oK1szo%2BnhcZUYuuAgrRlGBFoFpmUYk3bPJY%3D' (2025-12-01)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/741dabf48e7effc1cc6a43ed97d071214ed2cc31?narHash=sha256-tSbNnYn/gB6DDhZ8ursugXBIMxZ64zuveD517ZpXYZs%3D' (2025-12-01)
  → 'github:NuschtOS/nuschtpkgs/d0486baff9990ea5571d639888d48d82d0e48f66?narHash=sha256-%2BUXIW88fchu2tG1GZO8JZHku4w9ryFE0PQ80gvEmDig%3D' (2025-12-01)

SuperSandro2000 committed:
Fix typo

SuperSandro2000 committed:
Remove unused imports

SuperSandro2000 committed:
Format

SuperSandro2000 committed:
Do not route to options if disabled

SuperSandro2000 committed:
Try two parallel

SuperSandro2000 committed:
Force non interactive tty

SuperSandro2000 committed:
Fix new possibly undefined types

SuperSandro2000 committed:
Try one character grouping to speed things up

SuperSandro2000 committed:
Add WIP maintainer teams

SuperSandro2000 committed:
Only inherit what we need

SuperSandro2000 committed:
Add placeholder to ignore packages

SuperSandro2000 committed:
Fix license eval for none spdx licenses

SuperSandro2000 committed:
Fix maintainer eval

SuperSandro2000 committed:
Revert "Try one character grouping to speed things up"

This reverts commit 41c2af8da5624d4b1fd4e6cfbaf4a2c5f341455e.

SuperSandro2000 committed:
Unify mapping names

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/0b49397b3c1160944687aba813799f3f41fe6500?narHash=sha256-henej5X7oK1szo%2BnhcZUYuuAgrRlGBFoFpmUYk3bPJY%3D' (2025-12-01)
  → 'github:NuschtOS/ixx/4555316919dd9969ca1422a3686cbce580734084?narHash=sha256-grJXRU/aAMB1kkm/Yvf5I9tXOCClXsTBeBmDmr0pL%2BU%3D' (2025-12-02)

SuperSandro2000 committed:
Update fixx automatically

SuperSandro2000 committed:
Suppress rogue m being logged

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/4555316919dd9969ca1422a3686cbce580734084?narHash=sha256-grJXRU/aAMB1kkm/Yvf5I9tXOCClXsTBeBmDmr0pL%2BU%3D' (2025-12-02)
  → 'github:NuschtOS/ixx/d2fb751c99983a004a829859c320491e53441c1f?narHash=sha256-6J5OQrOiWqQdgAv/H5s%2BLx4exVNjhCH9/1PWWs3f1n0%3D' (2025-12-02)

SuperSandro2000 committed:
Move listPackages into extra file

SuperSandro2000 committed:
Fix warning

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/d2fb751c99983a004a829859c320491e53441c1f?narHash=sha256-6J5OQrOiWqQdgAv/H5s%2BLx4exVNjhCH9/1PWWs3f1n0%3D' (2025-12-02)
  → 'github:NuschtOS/ixx/e9273d0296e81326650a68699ffac1e999daf5d6?narHash=sha256-JxwUp1E41karGHgYoYa4yhamjiZAUX%2BeSE69dDZVWrQ%3D' (2025-12-02)

SuperSandro2000 committed:
Copy fixx

SuperSandro2000 committed:
Bring recursion back

SuperSandro2000 committed:
Dedupe teams and maintainers

SuperSandro2000 committed:
Fix teamMapping eval

SuperSandro2000 committed:
Fix for packages with not meta.team

SuperSandro2000 committed:
Bring down maintainers to just github ids again

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/d0486baff9990ea5571d639888d48d82d0e48f66?narHash=sha256-%2BUXIW88fchu2tG1GZO8JZHku4w9ryFE0PQ80gvEmDig%3D' (2025-12-01)
  → 'github:NuschtOS/nuschtpkgs/e0dd271390039ca3eaaa4b235d3ad60ca10204cc?narHash=sha256-mGI/9DHt3qxV7z3t5vDTzTPOhKdmiqGHOCjWWqHQtoU%3D' (2025-12-03)

SuperSandro2000 committed:
Fix team component

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/e9273d0296e81326650a68699ffac1e999daf5d6?narHash=sha256-JxwUp1E41karGHgYoYa4yhamjiZAUX%2BeSE69dDZVWrQ%3D' (2025-12-02)
  → 'github:NuschtOS/ixx/9731e0e146ed8a8d6c7a096549303dca9406b3f6?narHash=sha256-btRGO4VRZ%2BsFvyDjwjm8L%2BNUvxvQldTwbfW5H88g5pM%3D' (2025-12-04)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/e0dd271390039ca3eaaa4b235d3ad60ca10204cc?narHash=sha256-mGI/9DHt3qxV7z3t5vDTzTPOhKdmiqGHOCjWWqHQtoU%3D' (2025-12-03)
  → 'github:NuschtOS/nuschtpkgs/5b46383059db7a38b8cce25dde8cde7b88c858a9?narHash=sha256-c6C7aFhlarQDGAWelnDnrjHN%2BaNo4BpU6HnSjGS0%2BHA%3D' (2025-12-04)

SuperSandro2000 committed:
Rename meta directory to chunks

SuperSandro2000 committed:
Add fixx-dist-debug

SuperSandro2000 committed:
Make search appear more snappy by decreasing bounce time

SuperSandro2000 committed:
Update ixx, bump pnpm

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nix-index-database':
    'github:Mic92/nix-index-database/5a3ff8c1a09003f399f43d5742d893c0b1ab8af0?narHash=sha256-77jL5H5x51ksLiOUDjY0ZK8e2T4ZXLhj3ap8ETvknWI%3D' (2025-11-30)
  → 'github:Mic92/nix-index-database/82befcf7dc77c909b0f2a09f5da910ec95c5b78f?narHash=sha256-d3NBA9zEtBu2JFMnTBqWj7Tmi7R5OikoU2ycrdhQEws%3D' (2025-12-09)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/5b46383059db7a38b8cce25dde8cde7b88c858a9?narHash=sha256-c6C7aFhlarQDGAWelnDnrjHN%2BaNo4BpU6HnSjGS0%2BHA%3D' (2025-12-04)
  → 'github:NuschtOS/nuschtpkgs/167560714b5c24ff564d27cab5ec1a717971a5a1?narHash=sha256-movudj7psSLJlnmrhxGokSmPzAsJF85VakeGQk7loi4%3D' (2026-01-12)

MarcelCoding committed:
misc: make it build again

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
ci: redurce workers

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'ixx':
    'github:NuschtOS/ixx/203e61da5d176edd289693c7def98a8241139295?narHash=sha256-GXj%2BQzgdc53MB5n75ldEHuQ7pWwjBPq2pnimLL/o7PA%3D' (2025-12-05)
  → 'github:NuschtOS/ixx/6c74eb8eb1c78f4ef350173b77ac5145189d8e82?narHash=sha256-9XVmnC3sW/flvGCrG0d8Vo9Oe8jKOo6KgK%2B6v/xt1UI%3D' (2026-01-13)

MarcelCoding committed:
misc: update deps

MarcelCoding committed:
ci: build search data in seperate process

MarcelCoding committed:
ci: less paralell

SuperSandro2000 committed:
Bump

SuperSandro2000 committed:
Wait for index to be ready, remove some ternary

SuperSandro2000 committed:
We don't need to free

MarcelCoding committed:
misc: update ixx

MarcelCoding committed:
ci: dont pipe build output through cat

MarcelCoding committed:
ci: enable swap

MarcelCoding committed:
ci: prevent the wrangler action from installing all npm packages again...

MarcelCoding committed:
package: refactor package view; hide scopes if only one scope exists

SuperSandro2000 committed:
Add changelog, description

SuperSandro2000 committed:
Get a proper line when there is a parsing error

SuperSandro2000 committed:
Convert changelog to list

SuperSandro2000 committed:
Add sourceProvenance

SuperSandro2000 committed:
Format

SuperSandro2000 committed:
Make some more licenses optional

SuperSandro2000 committed:
Fall back to license input which might be a fullName

SuperSandro2000 committed:
Add unfree licenses to license map, add hover text about unfree and redistributable licenses

SuperSandro2000 committed:
Bump

MarcelCoding committed:
list-packages: exclude coqPackages

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/167560714b5c24ff564d27cab5ec1a717971a5a1?narHash=sha256-movudj7psSLJlnmrhxGokSmPzAsJF85VakeGQk7loi4%3D' (2026-01-12)
  → 'github:NuschtOS/nuschtpkgs/c4ed71cc1a397b5374cd5fcfea68e75662b08806?narHash=sha256-iydIExBTAY1nWygiioQ9bpyqB0NIs8SjuMQpFIdHNB4%3D' (2026-01-17)

MarcelCoding committed:
nix: use nix 2.33

SuperSandro2000 committed:
Move amount of options/packages down to Search label

SuperSandro2000 committed:
Rename options to results as it could also be a package

SuperSandro2000 committed:
Fix some types

SuperSandro2000 committed:
Linkify package types

SuperSandro2000 committed:
Do not return only scopeId

SuperSandro2000 committed:
Fix package link for overrides

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/c4ed71cc1a397b5374cd5fcfea68e75662b08806?narHash=sha256-iydIExBTAY1nWygiioQ9bpyqB0NIs8SjuMQpFIdHNB4%3D' (2026-01-17)
  → 'github:NuschtOS/nuschtpkgs/ec128cc9c2fc2e6677377e650dfe38c4e238aeff?narHash=sha256-tI1QHkne/uaS1SbdaNIH80OuHpavr9fV8JGatSKbLz4%3D' (2026-01-21)

MarcelCoding committed:
index: move meta from index to config

MarcelCoding committed:
scopes: only display if entries exist

MarcelCoding committed:
search: fix empty result list on page switch

MarcelCoding committed:
searcgh: cleanup

SuperSandro2000 committed:
Fix packages with none asci chars

SuperSandro2000 committed:
Also linkify null or package

SuperSandro2000 committed:
Only linkify packages when there are packages in scope

MarcelCoding committed:
options: fix pkgs link

MarcelCoding committed:
nix: exclude srcs

SuperSandro2000 committed:
flake: fix package eval

SuperSandro2000 committed:
Split nixpkgs specific package hacks off from other packages listings

SuperSandro2000 committed:
frontend: fix node_modules exclusion

SuperSandro2000 committed:
frontend: fix build when public/data exists from local testing

SuperSandro2000 committed:
wrapper: fix packagesEnabled condition

MarcelCoding committed:
nix: fix experimental features

MarcelCoding committed:
fix scope id handling; TODO: add url slug for scopes

MarcelCoding committed:
misc: update

MarcelCoding committed:
misc: cleanups

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
misc: regenerate without ssr

MarcelCoding committed:
misc: update header

MarcelCoding committed:
fixx: use npmjs.com version

MarcelCoding committed:
nix: update npm hash

MarcelCoding committed:
flake: match ixx version with fixx version

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
scopes: fix index

MarcelCoding committed:
nix: update to v2.34

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nix-index-database':
    'github:Mic92/nix-index-database/8fba98c80b48fa013820e0163c5096922fea4ddd?narHash=sha256-ZQ5z%2BfVhxYKtIFwtqGp5O0PD84BM1riASvqDaN5Xs%2Bs%3D' (2026-05-24)
  → 'github:Mic92/nix-index-database/1a2ea89c917781e88508d9fd2b507f2d2a0e173c?narHash=sha256-0BYqs8yKWkOz2Q7%2BSP18N5E5gmDKSo6LSxIVIa0wWes%3D' (2026-06-07)
• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/cc8d3defe8697c5325733952fdac1740d5d420a4?narHash=sha256-2aNlBiC63HeCxGzCaYU0UQ1CcKeNNwll9CoRAvw7Apk%3D' (2026-05-25)
  → 'github:NuschtOS/nuschtpkgs/331800de5053fcebacf6813adb5db9c9dca22a0c?narHash=sha256-x5UQuRsH3MqI0U9afaXSNqzTPSeZlRLvFAav2Ux1pNw%3D' (2026-05-31)

MarcelCoding committed:
list-packages: disable nixosTests

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
angular: remove hydration - ssr was removed before already

MarcelCoding committed:
search: forms expect a non null value, if the query param is not defined, this results in an error

MarcelCoding committed:
ui: fix scrollbar for search box

MarcelCoding committed:
a11y: apply some fixes

MarcelCoding committed:
misc: minor cleanups

MarcelCoding committed:
ci: use 26.05

MarcelCoding committed:
ui: fix scroll

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
ci: cleanup

MarcelCoding committed:
misc: fix hashes

MarcelCoding committed:
e2e: add package search tests

MarcelCoding committed:
search: fix missing name with single scope

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
flake: allow top-level functions to be overriden again

MarcelCoding committed:
nav: hide if only packages or options are available
